### PR TITLE
updated user agent string to match a more recent one

### PR DESCRIPTION
**Pull request by lasdkfj**
[(Im a noob) But I think it does sense to change the user agent as it makes them think it could be legit traffic by normal users. And avoids the servers to be IP Blocked because they have been identified to belong the same "botnet".](https://element.asra.gr/#/room/!lyMGtUfpDHiyyEDFeP:hackint.org/$k_f8su4RM-bb37OoBOJKX7MFLDWXrFvDkvqQQUTvMJI)

Maybe its a combination: if DatacenterIP+WeirdUserAgent+MuchTraffik > 3       -> Block 

Some tought in irc its the user agents causing the blocking others didnt. One said it would be easier to change IPs then the User Agent. IDK, this change does make sense imho. I didnt tested the codechange.

### CODE REVIEW

File: pipeline.py [hunk 1]
```diff
@@ -60,8 +60,8 @@
 #
 # Update this each time you make a non-cosmetic change.
 # It will be added to the WARC files and reported to the tracker.
-VERSION = '20230513.01'
-USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36'
```
Comment (Tristan971) [inline pipeline.py:64]: real browser seems to be using the full `(Windows NT 10.0; Win64; x64)` portion here tho

Comment (lasdkfj) [reply pipeline.py:64]: yes, you are right, I checked it on my Windows 10 Laptop. I copied it from https://www.whatismybrowser.com/guides/the-latest-user-agent/chrome

**Comment by Tristan971** [comment]
ping @JustAnotherArchivist c:

**Comment by lasdkfj** [comment]
People reported UA Blocks: https://matrix.to/#/!lyMGtUfpDHiyyEDFeP:hackint.org/$c8ao5GZ8atdp1aesZrJnNLk0TFV6Bk_8Pqek7igeeUA?via=hackint.org
But I dont know, if its still an issue. Others report to be running just fine.

**Comment by Arkiver2** [comment]
This is in since https://github.com/ArchiveTeam/imgur-grab/commit/24ac2aa220093ab649d262c35bacfc3bc5473b16.

**Pull request closed without merging**